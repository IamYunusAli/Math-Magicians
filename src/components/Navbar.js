import { NavLink } from 'react-router-dom';
import styles from './Navbar.module.css';

const Navbar = () => (
  <header className={styles.header}>
    <NavLink className={styles.htitle} to="/">
      Math Magicians
    </NavLink>
    <nav>
      <ul className={styles.nlink}>
        <li>
          <NavLink to="/">Home</NavLink>
        </li>
        <li>
          <NavLink to="/Calculator">Calculator</NavLink>
        </li>
        <li>
          <NavLink to="/Quote">Quote</NavLink>
        </li>
      </ul>
    </nav>
  </header>
);

export default Navbar;
