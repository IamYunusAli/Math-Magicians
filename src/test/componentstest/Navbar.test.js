import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../../components/Navbar';

test('should match Header snapshot', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
