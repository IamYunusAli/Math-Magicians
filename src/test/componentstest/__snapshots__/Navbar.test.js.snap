// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match Header snapshot 1`] = `
<header
  className="header"
>
  <a
    aria-current="page"
    className="htitle active"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <nav>
    <ul
      className="nlink"
    >
      <li>
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          className=""
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          className=""
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
